### PR TITLE
Pruned Websites and fixed HTML formatting

### PR DESCRIPTION
**Pull request by Wamy-Dev**
Deleted old and broken sites, as well as sites whose domains were not accessible.

I didn't add reasons, but I checked all of them myself, if you want you can check each one too. 

Did it myself from #105

### CODE REVIEW

File: showcase.html [hunk 1]
```diff
@@ -607,20 +417,6 @@ <h6>T Anime</h6>
                 <p>Anime Tracker</p>
             </div>
         </a>
-        <a class="item" href="https://github.com/sinkaroid/sinkaroid">
```
Comment (irfan-dahir) [inline showcase.html:610]: This one seems to be working

**Review by irfan-dahir — CHANGES_REQUESTED** [review]
Hi, thanks for taking the time to go through this

I've left one comment, other than that LGTM.

File: showcase.html (hunk 1 above)
Comment (Wamy-Dev) [reply showcase.html:610]: The website just leads to someone's personal profile. It's not a tool that anyone else can use which was my criteria. I guess it technically does work if you scroll to the very bottom and it shows his favorite anime which is gathered using Jikan.

Comment (irfan-dahir) [reply showcase.html:610]: Hmm, you're right.

**Pull request merged**